examples/algorithms/plot_snap.py

```diff
@@ -0,0 +1,43 @@
+"""
+==================
+SNAP Graph Summary
```

Comment:
Somewhere you should say that SNAP stands for:
(**S**ummarization by Grouping **N**odes on **A**ttributes and **P**airwise Relationships)

Also that it does not (in this case) stand for the Stanford Network Analysis Project (SNAP).

I was getting those confused.

Reply:
Makes sense.  Sorry for not clearing that up earlier.

I went ahead and added the distinction to both.